frontend/model/contracts/group.js

```diff
@@ -552,9 +552,7 @@ sbp('chelonia/defineContract', {
             allowPublicChannels: false
           },
           streaks: initGroupStreaks(),
-          profiles: {
-            [meta.username]: initGroupProfile(meta.createdDate)
-          },
+          profiles: {},
```

Comment:
Why was this removed?

Reply:
Because it's set when joining the group, which is now something that the group creator also does. (Also, because `meta.username` will be removed).